.gitignore

```diff
@@ -23,7 +23,7 @@ npm-error.log
 
 # Treeherder-specific
 _build/
-dist/
+build/
```

Comment:
We could still output to the `dist/` directory, but I slightly prefer `build/` and it's also the Neutrino default - so seemed preferable to just use the default instead.